### PR TITLE
Layout simplification, updated FAQ

### PR DESCRIPTION
**Pull request by tsa96**
Removes the hero section of the site and simplifies the FAQ layout, to deliberately look drier and less marketing-y.

Also updates FAQ content to use stuff in the google doc we were working on yesterday.

Desktop (1920 x 1080) and mobile (412 x 915) screenshots:
localhost_5173_(screen lol)
localhost_5173_(Pixel 7)

### CODE REVIEW

File: src/routes/+page.svelte [hunk 1]
```diff
@@ -53,214 +47,115 @@
...
+	<h2>Why not Source 2?</h2>
+	<p>
+		We do not have access to Source 2 and are not licensed to use Source 2. All of the games
+		currently using Strata are closely tied to existing Source 1 content, so Source 2 would be a
+		poor choice for us.
+	</p>
+	<p>
+		We do have some Source 2 features, such as the Panorama UI system, that were backported by Valve
+		to CS:GO. However, Strata is still almost entirely Source 1. Whilst Source 1 does contain
+		considerable tech-debt, we’ve been able to improve modernize it considerably, and we think its
```
Comment (JJL772) [inline src/routes/+page.svelte:97]:
```suggestion
		considerable tech-debt, we’ve been able to improve and modernize it considerably, and we think its
```

**Review by JJL772 — CHANGES_REQUESTED** [review]
Looks good besides a minor spelling mistake

File: src/routes/+page.svelte (hunk 1 above)
Comment (tsa96) [reply src/routes/+page.svelte]: Lol, this was like the one thing that I wrote. Fixed!

**Review by JJL772 — APPROVED** [review]
Awesome sauce

**Review by Gocnak — APPROVED** [review]

**Comment by tsa96** [comment]
@ozxybox this good with you? I know you've read the text but may as well get confirmation. Anyone can go ahead and merge after Ozxy's approval.

**Pull request merged**